src/protocol/thread/thread.ts

```diff
@@ -620,11 +620,16 @@ class _ThreadFront {
     return await pause.getScopes(frameId);
   }
 
-  async evaluate(asyncIndex: number, frameId: FrameId | undefined, text: string) {
+  async evaluate(
+    asyncIndex: number,
+    frameId: FrameId | undefined,
+    text: string,
+    pure: boolean = false
+  ) {
```

Comment:
Better to have an object as a single param that may have optional fields than doing a weird mix of optional params as `...arguments`.